fastatophy/fastatophy.go

```diff
@@ -0,0 +1,82 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Convert a multiple-sequence alignment in FASTA to
+// PHYLIP (sequential) format
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/biogo/biogo/alphabet"
+	"github.com/biogo/biogo/io/seqio"
+	"github.com/biogo/biogo/io/seqio/fasta"
+	"github.com/biogo/biogo/seq/linear"
+)
+
+func main() {
+	var (
+		inf            = flag.String("inf", "test.aln", "input FASTA filename")
+		outf           = flag.String("outf", "test.phy", "output PHYLIP filename")
+		help           = flag.Bool("help", false, "help prints this message.")
+		totSeq, alnLen int
+	)
+
+	flag.Parse()
+	if *help {
+		flag.Usage()
+		os.Exit(0)
+	}
+	// open input (first pass) to get totSeq and alnLen
+	in, err := os.Open(*inf)
+	if err != nil {
+		log.Fatalf("open FASTA file (1st pass): %v.\n", err)
+		os.Exit(1)
```

Comment:
Ahh, I see - the `os.Exit(1)` is dead, not the entire conditional. `log.Fatal*` calls `os.Exit(1)`.